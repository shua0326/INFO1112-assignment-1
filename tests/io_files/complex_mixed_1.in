ls .. | grep ".py$"
which ls grep
var old_path ${PATH}
var PATH ""
which ls grep
echo ${PATH}
var PATH ${old_path}
which ls grep
ls
