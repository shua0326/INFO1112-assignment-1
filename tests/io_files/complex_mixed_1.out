>> mysh.py
parsing.py
>> /usr/sbin/ls
/usr/sbin/grep
>> >> >> ls not found
grep not found
>> mysh: command not found: echo
>> >> /usr/sbin/ls
/usr/sbin/grep
>> io_files
run_tests.sh
words.txt
>> 
